### PR TITLE
Implement a fixed prepend for the SingleSelectedComponent's toggle button

### PR DESCRIPTION
**Pull request by gdostie**
I needed a way to prepend some text before the selected value of a SingleSelectedComponent, but I did not wanted this prepend to be shown on options in the dropdown.

### CODE REVIEW

File: src/components/select/SingleSelectConnected.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ export interface ISingleSelectOwnProps extends ISelectProps {
     toggleClasses?: string;
     onSelectOptionCallback?: (option: string) => void;
     items?: IItemBoxProps[];
+    togglePrepend?: React.ReactNode;
```
Comment (MichelML) [inline src/components/select/SingleSelectConnected.tsx]: why "toggle" prepend?

**Comment by pochretien** [comment]
you can add an example

**Comment by pochretien** [comment]
The actual prepend props can do the job to add a text I think

File: src/components/select/SingleSelectConnected.tsx (hunk 1 above)
Comment (GermainBergeron) [reply src/components/select/SingleSelectConnected.tsx]: The toggle is the button I guess, maybe buttonPrepend would be clearer

**Comment by gdostie** [comment]
@pochretien the prepend prop of each list item prepends the text in the dropdown values aswell which is not what I want, I want it to be displayed only in the button

**Pull request merged**